File: frontend/frontend/src/entities/MessageContent.ts
export class MessageContent {
    readonly id: number;
    readonly content: string;
    readonly messageThreadId: number;
  
    constructor(
      id: number,
      content: string,
      messageThreadId: number,
    ) {
      this.id = id;
      this.content = content;
      this.messageThreadId = messageThreadId;
    }
    getId(): number {
      return this.id;
    }
  
    getContent(): string {
      return this.content;
    }
  
    getMessageThreadId(): number {
      return this.messageThreadId;
    }
  }

File: backend/src/main/java/com/restful/api/model/MessageContent.java
package com.restful.api.model;

import javax.persistence.*;

@Entity
public class MessageContent {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    @Column
    private String content;
    @Column
    private int messageThreadId;

    public MessageContent(String content, int messageThreadId) {
        this.content = content;
        this.messageThreadId = messageThreadId;
    }

    public MessageContent() {

    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public int getMessageThreadId() {
        return messageThreadId;
    }

    public void setMessageThreadId(int messageThreadId) {
        this.messageThreadId = messageThreadId;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }
}
